packages/amplication-server/src/core/moduleAction/dto/EnumModuleActionRestVerb.ts

```diff
@@ -0,0 +1,16 @@
+import { registerEnumType } from "@nestjs/graphql";
+
+export enum EnumModuleActionRestVerb {
```

Comment:
also this, is on code-gen-types

Reply:
@barshimi No, this type is defined in the server. The code-gen-types use it from  his  model file